### ISSUE TITLE
Suggestion: Move the Web files to the coolprops wiki on github

### ISSUE THREAD

**Issue by lgpage** [ibell/coolprop#11]
As a suggestion I think it might be beneficial to move the Web files to the github wiki:
git@github.com:ibell/coolprop.wiki.git

That is if you want to migrate the sourceforge website to github.

**Comment by ibell** [comment]
I'm amenable, but only if we can get coolprop.sf.net to redirect to github.  What's the advantage of hosting on github?

**Comment by ibell** [comment]
Any further thoughts on this one?

**Comment by thorade** [comment]
One reason to move the web files to github would be to have everything in one place.
Cleanest solution in the long term would be to register a domain like coolprop.org and let it point to whereever you want. 
On github, every project can have a wiki, but also normal webpages, see http://pages.github.com/
The address for these pages would then be something like http://ibell.github.io/coolprop/

**Comment by ibell** [comment]
The only problem I see with this idea is that we use Sphinx to generate the
docs, do some documentation buildtime validation, etc.  Which would not be
possible to do if we used markdown directly.  Or we could just host the
pages on github, which could work too.  The doxygen files are pretty heavy
though...

I totally see the appeal of doing it all in markdown, but I wonder whether
we couldn't just have scripts to generate the necessary data in the
documentation and partly do away with Sphinx.  That is one idea, and if
someone wanted to take it on, I'm game. What would be really great is if
you could get Sphinx to spit out markdown, which we could then just keep on
the web. @jowr, do you have feelings about this?

**Comment by jowr** [comment]
No, no strong feelings. A short tour on the net showed that Sphinx can
produce Markdown:
http://stackoverflow.com/questions/13396856/markdown-output-for-sphinx-based-documentation
http://johnmacfarlane.net/pandoc/

I think my preferred solution would be to have a domain and host the
pages on github. This would also make things easier in terms of support.
A shared email address or even a ticket handling system could definitely
improve the support experience and remove some of your workload, Ian.

**Comment by jowr** [comment]
It looks like Rod is going to handle the code documentation. For a general purpose homepage with more examples and additional downloads, we might still consider github pages:
http://24ways.org/2013/get-started-with-github-pages/

**Comment by Rod-Persky** [comment]
I've gotten the docs to build nicely on readthedocs, I've just got to pull it into the main repo.

**Comment by jowr** [comment]
:+1:  for that...

**Comment by ibell** [comment]
So the one bit left then is to generate the dynamic examples and fluid information.  Anyone have an idea how best to handle that?  There's quite a lot of data associated with one rebuild of the website, on the order of 100 MB if I remember correctly including all the figures, doxygen code, etc.  We can't really commit it because then we will end up with enormous git repos.

But I really like the idea of getting away from Sphinx if we can.  The nice parts about sphinx are: 
- automatic source code documentation
- plotting
- cross-referencing, automatic header pages, etc.

Maintaining large Sphinx projects is quite a pain.  And it makes it hard for everyone to contribute; it would be much easier if we could just edit Markdown or RST or HTML files.

@Rod-Persky  your work on RTD is greatly appreciated! Let's add a link to the main page, want to do that?

**Comment by Rod-Persky** [comment]
Sorry I haven't been able to continue working on the RTD for a bit, I should have time after Feb 5. Also on a related note, how do you want coolprop referenced? I'm thinking of just referencing the ASME ORC poster, is there a preferred method @ibell

**Comment by jowr** [comment]
Have a look at http://pubs.acs.org/doi/abs/10.1021/ie4033999. Full bibliographic information should follow soon. I currently use this BibTex entry:

```
@article{CoolPropACS,
author = {Bell, Ian and Wronski, Jorrit and Quoilin, Sylvain and Lemort, Vincent},
title = {Pure- and Pseudo-Pure Fluid Thermophysical Property Evaluation and the Open-Source Thermophysical Property Library CoolProp},
journal = {Industrial \& Engineering Chemistry Research},
volume = {0},
number = {ja},
pages = {null},
year = {2014},
doi = {10.1021/ie4033999},
URL = {http://pubs.acs.org/doi/abs/10.1021/ie4033999},
eprint = {http://pubs.acs.org/doi/pdf/10.1021/ie4033999}
}
```

I hope this is what you are looking for.
